fab/inventory/l10k

```diff
@@ -0,0 +1,61 @@
+[l10kserver1]
+10.162.36.254
+
+#[l10kbackup1]
+#ip_backup_server
```

Comment:
one thing i was thinking about for the backup server is that i'm not sure if we'll have an internal vpn for the two servers. we breaks a lot of the assumptions in the ansible code. for example if we streamed postgres data to the backup server, it'd be in plain text. something we should probably think about/discuss